www/src/App.tsx

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
+import { GetTokenSilentlyOptions, useAuth0 } from '@auth0/auth0-react';
+
+import AuthenticateButton from './components/auth/AuthenticateButton';
 import logo from './logo.svg';
 import './App.css';
 
+// TODO: Remove dummy requests
+type tokenAcquierer = (options?: GetTokenSilentlyOptions | undefined) => Promise<string>;
```

Comment:
Isn't the `| undefined` implicit to `options?`?

Reply:
It should be, it was the type that came up when I hovered over `getAccessTokenSilently` but I'll see if I can remove it